setup.py

```diff
@@ -56,17 +57,32 @@ def extensions():
     """ setup extensions for this module
     """
     exts = []
-    exts.append(
-        Extension(
-            'pytng.pytng',
-            sources=glob('pytng/src/compression/*.c') + glob(
-                'pytng/src/lib/*.c') + glob('pytng/src/external/*.c') + ['pytng/pytng.pyx'],
-            include_dirs=[
-                "pytng/include/", "pytng/src/external/", "{}/include".format(sys.prefix),
-                np.get_include()
-            ],
-            library_dirs=["{}/lib".format(sys.prefix)],
-            libraries=['z'], ))
+    if sys.platform.startswith("win"):
```

Comment:
Could we have the Extension defined once and the differences to DEF and libraries put behind the startswith condition?

Reply:
probably... do know anything about windows... I'm a bit stumped here.